src/components/DateRangePicker.tsx

```diff
@@ -78,6 +78,7 @@ export const DateRangePicker = ({ dateRangeSelector }: Props) => {
         <div className={`flex flex-row ${width && width < 600 ? 'flex-wrap mt-2 mb-2 ml-0' : 'ml-2'}`}>
           <LocalizationProvider dateAdapter={AdapterDateFns}>
             <DatePicker
+              inputFormat='MM-dd-yyyy'
```

Comment:
@anastasia-escher, ISO 8601 (https://www.iso.org/iso-8601-date-and-time-format.html) defines the date format as `yyyy-MM-dd` ;)